### ISSUE TITLE
How to play separately? So not synchronized?

### ISSUE THREAD

**Issue by ThaDaVos** [Saiyato/volumio-snapcast-plugin#52]
I've got Synchronized multiroom working but was wondering - how can I temporarily disconnect a client or something so it can play separately?

**Comment by Saiyato** [comment]
You can stop the plugin, or disable the client in the plugin itself.
The client doesn't need any config overrides, so you should be fine by just disabling it.

**Comment by ThaDaVos** [comment]
How do you stop the plugin?
Just through the plugins list?

**Comment by ThaDaVos** [comment]
Would love if you just press the play button on a client it would "temporarily" disconnect the stream for local play

**Comment by Saiyato** [comment]
Yes, you can stop it from plugin management.
As for the request, I agree that would be nice, but we need to functionally design this, before moving to coding the functionality.

**Comment by ThaDaVos** [comment]
Ah I understand - currently just disabling the full plugin

Also going to open another issue regarding lot's of noise on the Pine64 board when using this plugin.
See https://github.com/Saiyato/volumio-snapcast-plugin/issues/53

**Comment by andrewj15** [comment]
I'm wondering how to do this myself as have tried to turn off the server and the client from within the plugin as well as disabling the plugin and neither work. When I do this each time i try with each option attempted separately i get no audio through my system. Note i am trying this only on my snapcast server Raspberry PI 3. 

My goal is to be able to have the option to play hi-res files on one of my devices, but also allow it to connect to snapcast when i want to play multiroon sync and accept the lower sample rate. Are there any workarounds for this configuration?

**Comment by Saiyato** [comment]
I see, if you want to _not_ play through the server, you might want to re-enable playback for MPD in the mpd.conf (either over the terminal or via the settings of the plugin). You can even enable both ALSA and fifo if you want.

Have you read the snapcast wiki for configuration? That might help give you an idea as to how you can configure it.

**Comment by andrewj15** [comment]
Thanks, yeah Ive read through the wiki's and snapcast github but it's not that clear. 

I think an option may be to swap my snapcast server from my current config.

If I set my current server to just be a client, so that if I just play locally on the client only it shouldn't be using snapcast to adjust the sample rate.

**Comment by andrewj15** [comment]
Yep that got it working. I have my secondary volumio setup as the Server, and use my main volumio as the client. By setting up my main volumio as the client I'm able to run direct to the client and not use snapcast which then allows me to play higher sample rates without being resampled.

**Comment by Saiyato** [comment]
Yes, one of the snap-components will have changed config files (mpd.conf/asound.conf), which can impact quality indeed.
